deploy/charts/csi-driver/values.yaml

```diff
@@ -50,24 +53,43 @@ app:
   # -- Overrides path to root kubelet directory in case of a non-standard k8s install.
   kubeletRootDir: /var/lib/kubelet
 
+# -- Optional additional annotations to add to the csi-driver DaemonSet
+# daemonSetAnnotations: {}
+
+# -- Optional additional annotations to add to the csi-driver Pods
+# podAnnotations: {}
```

Comment:
@MaesterZ do you know why these options are missing from the README.md file?

Reply:
Not sure how the documentation generator works, it's because they are commented out?

Reply:
maybe? Is this something you can/ want to investigate?

Reply:
I uncommented the values, works out.